### PR TITLE
Spanish translation. Issue #235

### LINKED ISSUE #235: Spanish translation
### Prerequisites

- [x] I'm using latest version: v 0.16.0

### Description

Hello. I want colaborate translating stretchly app to spanish.

Operating system: Debian

### PR DESCRIPTION
**Pull request by manueljaguilera**
Issue: fixes #235 

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [x]  issue was opened to discuss proposed changes before starting implementation.
- [ ]  during development, `node` version specified in `package.json` was used.
- [x]  package versions were not changed.
- [x]  app version number was not changed.
- [ ]  all new code has tests to ensure against regressions.
- [ ] `npm run lint` reports no offenses.
- [ ] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.

### Description of the Change

First spanish translation.

### Verification Process

Not needed.

### CODE REVIEW

File: app/locales/es.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  },
+  "settings2": {
+    "settings": "Configuración",
+    "audioScheme": "Esquema de sonido",
+    "crystalGlass": "crystal glass",
+    "windChime": "wind chime",
+    "ticToc": "tic toc",
+    "silence": "silencio",
+    "colorScheme": "Esquema de color",
+    "greenClouds": "green <br/> clouds",
```
Comment (hovancik) [inline app/locales/es.json]: you can be creative and use some other names ;] eg. when I was translating for Slovak language, I did not translate word by word

File: README.md [hunk 2]
```diff
@@ -153,6 +153,7 @@ Before implementing a feature, please open an Issue first, so we can be sure tha
 - Nic Desjardins, @nicdesjardins
 - Vladislav Kuznecov, @fizvlad
 - Oleg V., @neooleg
+- Manuel Jesús Aguilera Castro, @manueljaguilera
```
Comment (hovancik) [inline README.md]: Could you please also update CHANGELOG.md and settings3.html?  

in settings3.html you need to add something like this:  `<option value="fr">Français</option>`

**Comment by hovancik** [comment]
@manueljaguilera hey, do you plan to implement changes I've requested?

**Comment by manueljaguilera** [comment]
Hi! I feel the wait but these days I'm very busy. If you don't mind, I will
work on the changes in a few weeks.


Manuel Jesús Aguilera Castro

Teléfono 658 368 111

**Pull request merged**